pom.xml

```diff
@@ -215,6 +215,9 @@
 				<configuration>
 					<!-- The package of your generated sources -->
 					<packageName>tds.exam.results.trt</packageName>
+                    <sources>
+                        <source>src/main/resources/xsd</source>
```

Comment:
Yes, moving it to src/main/resources does add it to the classpath.  However, the plugin looks at `src/main/xsd` so I have to tell the plugin where to find the xsd.

@malaffoon - Currently the file isn't on the classpath, and this change fixes that